pulumi/infra/config.py

```diff
@@ -234,3 +235,11 @@ def container_repository() -> Optional[str]:
     if repo_name:
         return f"docker.cloudsmith.io/{repo_name}"
     return None
+
+
+# This should be equivalent to what HOST_IP_IN_NOMAD resolves to. However, we need this for the consul proxy defaults,
+# so we can't use var interpolation. This should be temporary until we get consul dns working within containers
+def get_nomad_ip() -> str:
```

Comment:
I forgot that this runs inside a docker container so this won't work. It will need to be set manually